.github/renovate.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["config:js-app", "schedule:weekly"],
```

Comment:
We should use `config:js-lib` instead of `config:js-app`

Reply:
fixed :)